org.eclipse.lemminx/src/main/resources/org/eclipse/lemminx/services/snippets/doctype-snippets.json

```diff
@@ -0,0 +1,58 @@
+{
+  "Insert SYSTEM DOCTYPE": {
+    "prefix": [
+      "<!DOCTYPE"
+    ],
+    "suffix": ">",
+    "body": [
+      "<!DOCTYPE ${1:root-element} SYSTEM \"${2:file}.dtd\" >"
+    ],
+    "description": "Insert SYSTEM DOCTYPE"
+  },
+  "Insert PUBLIC DOCTYPE": {
+    "prefix": [
+      "<!DOCTYPE"
+    ],
+    "suffix": ">",
+    "body": [
+      "<!DOCTYPE ${1:root-element} PUBLIC \"${2:public-id}\" \"${3:file}.dtd\" >"

```

Comment:
Remove extra space before the closing `>`

Reply:
fixed